packages/aragon-wrapper/src/cache/index.js

```diff
@@ -41,6 +41,16 @@ export default class Cache {
     }
   }
 
+  /**
+   * Get a standard cache key
+   *
+   * @param {string} proxyAddress
+   * @param {string} location
+   */
+  getCacheKey(proxyAddress, location) {
```

Comment:
Nitpick: wondering if this would fit better outside of the `Cache` class since it's not really related to any `Cache` instances?

Reply:
Yeah I kind of felt that too